consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/network/IbftMessageTransmitter.java

```diff
@@ -50,7 +52,7 @@ public void multicastProposal(final ConsensusRoundIdentifier roundIdentifier, fi
 
     final ProposalMessageData message = ProposalMessageData.create(signedPayload);
 
-    multicaster.send(message);
```

Comment:
Look like Mulitcaster.send(MessageData message) can be removed now since all messages are going through the gossiper.